include/classes/DAO/MyownPatchesFilesDAO.class.php

```diff
@@ -61,7 +60,7 @@ public function Create($myown_patch_id, $action, $name, $location,
 		}
 		else
 		{
-			return mysql_insert_id();
+			$this->insertID();
```

Comment:
Could you use `getInsertID` rather than `insertID`? Thanks.

Reply:
Would do that